packages/components/src/__future__/Tag/_docs/Tag.stories.tsx

```diff
@@ -1,94 +1,52 @@
-)
+export const Playground: StoryObj = {
+  render: args => <Tag {...args}>My tag</Tag>,
+}
+
+export const SimpleTag: StoryObj = {
+  render: args => <Tag {...args}>My Tag</Tag>,
+}
 
-type ListProps = {
-  children: React.ReactNode
+export const Children: StoryObj = {
+  render: args => <Tag {...args}>This text is the children</Tag>,
 }
-const List = ({ children }: ListProps): JSX.Element => (
-  <div className="flex flex-col gap-12 items-start">{children}</div>
-)
 
-const StickerSheetTemplate: StoryFn = () => (
-  <StickerSheet heading="Tag">
-    <StickerSheet.Header headings={["Label Only", "Icon"]} />
-    <StickerSheet.Body>
-      <StickerSheet.Row>
-        <List>
-          <Tag>Gray</Tag>
-          <Tag color="blue">Blue</Tag>
-          <Tag color="green">Green</Tag>
-          <Tag color="yellow">Yellow</Tag>
-          <Tag color="orange">Orange</Tag>
-          <Tag color="red">Red</Tag>
-          <Tag color="purple">Purple</Tag>
-        </List>
-        <List>
-          <Tag Icon={<TagIcon role="presentation" />}>Gray</Tag>
-          <Tag color="blue" Icon={<TagIcon role="presentation" />}>
-            Blue
-          </Tag>
-          <Tag color="green" Icon={<TagIcon role="presentation" />}>
-            Green
-          </Tag>
-          <Tag color="yellow" Icon={<TagIcon role="presentation" />}>
-            Yellow
-          </Tag>
-          <Tag color="orange" Icon={<TagIcon role="presentation" />}>
-            Orange
-          </Tag>
-          <Tag color="red" Icon={<TagIcon role="presentation" />}>
-            Red
-          </Tag>
-          <Tag color="purple" Icon={<TagIcon role="presentation" />}>
-            Purple
-          </Tag>
-        </List>
-      </StickerSheet.Row>
-    </StickerSheet.Body>
-  </StickerSheet>
-)
+export const Color: StoryObj = {
+  render: _args => (
+    <>
+      {TagColorKeys.map(color => (
+        <Tag color={color} icon={<TagIcon role="presentation" />} key={color}>
+          {color}
+        </Tag>
+      ))}
+    </>
+  ),
+}
 
-export const StickerSheetDefault = StickerSheetTemplate.bind({})
-StickerSheetDefault.storyName = "Sticker Sheet (Default)"
-StickerSheetDefault.parameters = {
-  chromatic: { disable: false },
-  controls: { disable: true },
+export const Icon: StoryObj = {
+  render: _args => (
+    <>
+      <Tag icon={<AcademyIcon role="presentation" />}>AcademyIcon</Tag>
+      <Tag color="yellow" icon={<ActionOffIcon role="presentation" />}>
+        ActionOffIcon
+      </Tag>
+      <Tag color="green" icon={<AddIcon role="presentation" />}>
+        AddIcon
+      </Tag>
+    </>
+  ),
 }
 
-export const StickerSheetRTL = StickerSheetTemplate.bind({})
-StickerSheetRTL.storyName = "Sticker Sheet (RTL)"
-StickerSheetRTL.parameters = {
-  chromatic: { disable: false },
-  controls: { disable: true },
-  textDirection: "rtl",
+export const ClassNameOverride: StoryObj = {
+  render: _args => <Tag classNameOverride={styles.myCustomClass}>Hello</Tag>,
```

Comment:
Use `args` key instead :)

Given this is documentation only, you can use Tailwind instead of defining a scss class.